foglamp.version

```diff
@@ -1 +1 @@
-foglamp_version>=1.6
+foglamp_version>=1.7
```

Comment:
@Singhal-Vaibhav  Missing `service_notification.version` only. Please add.

Here we have `foglamp.version` as well, is this really required @pintomax  @MarkRiddoch

Reply:
@praveen-garg  foglamp.version is required for any plugin